src/partials/header-content.hbs

```diff
@@ -3,45 +3,40 @@
     <div class="navbar-brand">
       {{!-- <a class="navbar-item" href="{{{or site.url (or siteRootUrl siteRootPath)}}}"> --}}
       <a class="navbar-item" href="/">
+        <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20230926084108/Logo_FullColor_RGB_TransBG.svg" alt="{{ site.title }}" />
         {{!-- <link rel="icon" href="{{{uiRootPath}}}/img/favicon.ico" type="image/x-icon"> --}}
         {{!-- <img class="navbar-logo" src="{{{uiRootPath}}}/img/logo.svg" alt="{{ site.title }}" /> --}}
+      </a>
 
       {{#if (eq page.attributes.theme "docs") }}
-      <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20230926084108/Logo_FullColor_RGB_TransBG.svg" alt="{{ site.title }}" />
       <a href="/docs/" class="navbar-item no-left-padding" aria-label="Docs">
         Docs
       </a>
       {{else if (eq page.attributes.theme "cheat-sheet")}}
-      <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20230926084108/Logo_FullColor_RGB_TransBG.svg" alt="{{ site.title }}" />
       <a href="/docs/cypher-cheat-sheet/" class="navbar-item no-left-padding" aria-label="Labs">
         Cypher Cheat Sheet
       </a>
       {{else if (eq page.attributes.theme "labs")}}
-      <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20230926084108/Logo_FullColor_RGB_TransBG.svg" alt="{{ site.title }}" />
       <a href="/labs/" class="navbar-item no-left-padding" aria-label="Labs">
         Labs
       </a>
       {{else if (eq page.attributes.theme "kb")}}
-      <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20231023102534/neo4j-logo.svg" alt="{{ site.title }}" />
       {{!-- <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20230926084108/Logo_FullColor_RGB_TransBG.svg" alt="{{ site.title }}" /> --}}
       <a href="/developer/kb/" class="navbar-item no-left-padding page-name-white" aria-label="Knowledge Base">
         Knowledge Base
       </a>
       {{else if (eq page.attributes.theme "training")}}
-      <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20231023102534/neo4j-logo.svg" alt="{{ site.title }}" />
       {{!-- <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20230926084108/Logo_FullColor_RGB_TransBG.svg" alt="{{ site.title }}" /> --}}
       <a href="/graphacademy/online-training/" class="navbar-item no-left-padding page-name-white" aria-label="GraphAcademy">
         GraphAcademy
       </a>
       {{else if (eq page.attributes.theme "graphgists")}}
-      <img class="navbar-logo" src="https://dist.neo4j.com/wp-content/uploads/20231023102534/neo4j-logo.svg" alt="{{ site.title }}" />
```

Comment:
Content is still published to with themes that have a dark header background so we do still need to use this image

eg

- https://neo4j.com/graphgists/ (graphgists theme)
- https://neo4j.com/developer/genai-ecosystem/ (developer theme)
- https://neo4j.com/developer/kb (kb theme)

Reply:
Ouch, thanks. The commented out lines probably confused me.

Reply:
Yeah, all the different themes and layouts are confusing for sure. There's probably files in there that should be deleted as well, but I keep thinking - is it worth the effort, or are we better off starting from scratch with a new version?